<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1138</width>
    <height>662</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>1205</width>
    <height>921</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>1081</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="trainLayout">
    <item>
     <widget class="QPushButton" name="trainButton">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Start!</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_9">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>100</y>
     <width>531</width>
     <height>541</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="HOG_PCA_AdaBoost__Classifier_Label">
    <item>
     <widget class="QLabel" name="judul_hogpcaAdaBoost">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>HOG + PCA + AdaBoost Classifier</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="param_hogpcaAdaBoost">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="hasil_hogpcaAdaBoost">
      <property name="font">
       <font>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="2" column="0">
       <widget class="QLabel" name="hogView_hogpcaAdaBoost">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>158</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pcaButton_hogpcaAdaBoost">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>PCA</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QListWidget" name="faceClassList_hogpcaAdaBoost">
        <property name="minimumSize">
         <size>
          <width>158</width>
          <height>0</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QListWidget" name="pcaViewList_hogpcaAdaBoost">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>158</width>
          <height>0</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="hogButton_hogpcaAdaBoost">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>HOG</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="adaboostButton_hogpcaAdaBoost">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>AdaBoost</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="loadImageLayout_2">
          <item>
           <widget class="QPushButton" name="loadImageButton_hogpcaAdaBoost">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Load Image</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="loadImageView_hogpcaAdaBoost">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>200</height>
             </size>
            </property>
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <layout class="QVBoxLayout" name="preProcessingLayout_2">
          <item>
           <widget class="QPushButton" name="preProcessingButton_hogpcaAdaBoost">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>PreProcessing</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="preProcessingView_hogpcaAdaBoost">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>200</height>
             </size>
            </property>
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>531</width>
     <height>541</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="HOG_AdaBoost__Classifier_Label">
    <item>
     <widget class="QLabel" name="judul_hogAdaBoost">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="midLineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string>HOG + AdaBoost Classifier</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="param_hogAdaBoost">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="hasil_hogAdaBoost">
      <property name="font">
       <font>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="loadImageLayout">
        <item>
         <widget class="QPushButton" name="loadImageButton_hogAdaBoost">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Load Image</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="loadImageView_hogAdaBoost">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>200</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>200</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="hogButton_hogAdaBoost">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>HOG</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="hogView_hogAdaBoost">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="preProcessingLayout">
        <item>
         <widget class="QPushButton" name="preProcessingButton_hogAdaBoost">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>PreProcessing</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="preProcessingView_hogAdaBoost">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>200</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>200</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="adaboostButton_hogAdaBoost">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>AdaBoost</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="faceClassList_hogAdaBoost">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="datasetComboBox">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>60</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
